draft-ietf-oauth-v2-1.md

```diff
@@ -2227,26 +2213,12 @@ audience), typically a single resource server (or a list of resource
 servers), in the token.  Restricting the use of the token to a
 specific scope is also RECOMMENDED.
 
-The authorization server MUST implement TLS.  Which version(s) ought
-to be implemented will vary over time and will depend on the
-widespread deployment and known security vulnerabilities at the time
-of implementation.
-
-To protect against token disclosure, confidentiality protection MUST
-be applied using TLS with a ciphersuite that provides
-confidentiality and integrity protection.  This requires that the
-communication interaction between the client and the authorization
-server, as well as the interaction between the client and the
-resource server, utilize confidentiality and integrity protection.
-Since TLS is mandatory to implement and to use with this
-specification, it is the preferred approach for preventing token
-disclosure via the communication channel.  For those cases where the
-client is prevented from observing the contents of the token, token
-encryption MUST be applied in addition to the usage of TLS
-protection.  As a further defense against token disclosure, the
-client MUST validate the TLS certificate chain when making requests
-to protected resources, including checking the Certificate Revocation
-List (CRL) {{RFC5280}}.
+Risks related to token disclosure are mitigated by the mandatory
+usage of mechanisms providing
+confidentiality, authentication and integrity such as {{RFC8446}}
+(see {{tls-version}}). 
+if the client should be prevented from observing the contents of the token,
+token MUST be encrypted.
```

Comment:
@aaronpk not sure that an use-case requirement should use normative language (MUST)...